djstripe/models.py

```diff
@@ -459,7 +461,7 @@ def sync_current_subscription(self, cu=None):
         if sub:
```

Comment:
I know this isn't part of your changeset, but I think this variable should be expanded to stripe_subscription